public/style.css

```diff
@@ -110,3 +110,35 @@ button.ui.toggle.button.player-button + button.ui.toggle.button.player-button {
 button.ui.toggle.button.card-button + button.ui.toggle.button.card-button {
   margin-left: 10px;
 }
+
+/* DiseaseCube */
+img.leaflet-marker-icon.disease-cube-0.leaflet-zoom-animated.leaflet-interactive {
+  margin-left: 5px !important;
```

Comment:
Is there no better way to do this besides using the !important tab...

Reply:
I used !important to overwrite the inline styling created by leaflet. Can you think of another way?

Reply:
Oooohhh I see... can you not do inline styling here? & also merge with master plz!

Reply:
Nope since the Marker leaflet component doesn't take in styling. Unless there's a way to do that?